api_docgen/Format_tutorial.mld

```diff
@@ -247,6 +247,24 @@ break hint that precedes a closing parenthesis can show the boxes structure, if
 it leads to a new line; hence "\[(---\[(----\[(---b)\]b)\]b)\]" is printed like
 this:
 
+{[
+(---
+ (----
+  (---
+   )
+  )
+ )
+]}
+
+We can line up the opening and closing parenthesis vertically in two ways:
```

Comment:
We might want to point out that the closing parenthesis is too far on the right due to the indentation before explaining how to avoid this misalignment.

Reply:
I've inserted a new commit ` doc(format): explain why the parenthesis are not lined up vertically`

>Each break hint increases indentation by one, both for opening and closing
>parenthesis, because the parenthesis are just characters and don't have special
>meaning for the formatter.
>Notice that the closing parenthesis is not vertically aligned with the opening parenthesis.

Maybe that repeats information that should already be obvious (the formatter only knows about boxes and break hints, so of course it makes no distinction between opening and closing parenthesis, they're just characters), but we're in the tutorial here, and explaining why you got the output might help the reader "connect the dots" on how Format works.

I've also added another commit at the beginning:
`doc(format): the example uses 'open_box 1',  `open_hovbox 1'`

It is quite obvious that the box needs to have an indentation of 1, otherwise the opening parenthesis wouldn't be indented either, but make this explicit